schemas/vscs.json

```diff
@@ -216,6 +216,26 @@
                 "sectionEnd": "Bridges and Private IP",
                 "propertyOrder": 220
             },
+            "route_reflector_present": {
```

Comment:
We don't need the boolean. If the RR IP list is defined, the boolean can be assumed to be true.